src/index.js

```diff
@@ -19,6 +19,7 @@ server.get("/:uname", async (request, reply) => {
     reply.send(result.rows);
   }
 
+
```

Comment:
why this line?

Reply:
Without it the request only outputs the last line in my database.